be/src/runtime/exec_env.cpp

```diff
@@ -198,6 +198,8 @@ Status GlobalEnv::_init_mem_tracker() {
             calc_max_query_memory(_process_mem_tracker->limit(), config::query_max_memory_limit_percent);
     _query_pool_mem_tracker =
             regist_tracker(MemTracker::QUERY_POOL, query_pool_mem_limit, "query_pool", this->process_mem_tracker());
+    int64_t query_pool_spill_limit = query_pool_mem_limit * config::query_pool_spill_mem_limit_threshold;
```

Comment:
better have a check to the configvar, in case the value >1.0 or <0, what will be?

Reply:
yes, it's nice to have, but it will only happen if user abuses.